gelf/message_test.go

```diff
@@ -27,6 +29,14 @@ func TestInvalidFieldNames(t *testing.T) {
 	testMessage.Send()
 }
 
+func TestTimestamp(t *testing.T) {
+	now := float64(time.Now().Unix())
+	testMessage := NewMessage(LevelInfo, "This is a short test message.", "This is a long test message.")
+	if math.Abs(testMessage.Timestamp-now) > 1 {
```

Comment:
fair. I just didn't like the idea of an arbitrary maximum. HAVE YOU HEARD ABOUT THIS Y2K THING? I guess if gogelf is still being used in the year 2286 they can deal with it then